lavamoat/build-system/policy.json

```diff
@@ -2976,6 +2942,13 @@
         "debounce-stream>through": true
       }
     },
+    "@lavamoat/lavapack>readable-stream>abort-controller>event-target-shim": {
+      "globals": {
+        "Event": true,
+        "EventTarget": true,
+        "console": true
+      }
```

Comment:
We could ban these in overrides, this package is trying to shim them if not there and they're definitely there.